functions/interface.php

```diff
@@ -107,7 +106,7 @@ function admin_notices() {
 
 				<?php foreach ( $notices['server_config'] as $notice ) : ?>
 
-					<li><?php echo wp_kses_data( $notice ); ?></li>
+					<li><?php echo print_whitelist_html( $notice ); ?></li>
```

Comment:
It sounds like you want to output essentially arbitrary HTML here, in which case, there's not a really good option. Passing `'comment'` as the third param will give you just tags allowed in comments, which is generally safe. However, it doesn't allow anything not in that list, obviously.

Late escaping is about trying to avoid outputting unsafe code, but sometimes you want the ability to output arbitrary HTML. It seems like in this case, you want to do that, so just `echo` would probably be fine.

If you've already made sure the variable is safe before it gets here but phpcs is throwing an error, you can "open the escape hatch" and add a `// XSS OK` comment just before the `?>`. This tells phpcs that you're sure there's no XSS possibilities here and you're safe. Be very careful doing this though. :)